server/src/main/java/com/skillmagnet/User/User.java

```diff
@@ -36,6 +37,7 @@ public class User {
 
     private LocalDateTime lastLogin;
 
+    @Hidden
```

Comment:
Added this annotation, it removes it from the API docs since the FE shouldn't be supplying it when creating a new user. **Think we should open a PR to add these in more places, or find a better way of doing it**, eg generated id's and datestamps, other join columns. @nickerick 

Didn't think I should spam them all into this PR

**Before**
![beforeHidden](https://github.com/nickerick/skill-magnet/assets/72476187/ebd57d86-5ae9-402d-a18d-bd20961b2859)

**After**
![afterHidden](https://github.com/nickerick/skill-magnet/assets/72476187/c95b190f-804c-4e96-949b-ef7583c505e2)

when u click "try it out", it still includes the id and lastlogin which we generate automatically server side.

another example In course currently
![image](https://github.com/nickerick/skill-magnet/assets/72476187/852adece-e94c-4a4f-8f26-28fc3a1b355c)

Reply:
I'm in agreement to hide lessons from the user object - neat annotation

Reply:
Yeah that API spec right there is why it's better to create specific `@RequestBody` objects like you did with `EnrollsRequest`. Other than jakarta validation benefits, cleaning up the API spec is my main goal with doing that.

We just need to replace this with a `UserRequest` object

Reply:
Issue with you throwing that `@Hidden` on something like Id is yeah it cleans up our API spec for expected request body, but then it now also hides ID when we try `GET /user/{uid}`